### ISSUE TITLE
top-5 classes showing in Opera, not Firefox

### ISSUE THREAD

**Issue by victoriastuart** [transcranial/keras-js#9]
First, very cool - great work!  :-D

For your reference, keras-js runs fast in Opera; appears to run (slower) in Firefox, but does not display the top-5 classes (50-layer ResNet tried; supplied wolf image).

Firefox 49.0.1 - lots of plugins [re: this issue, disabled AdBlockPlus; uBlock Origin; Ghostery]
Opera 40.0 - minimal plugins
Arch Linux x86_64; Intel Core i7-4790 CPU; ...

Private message me -- if desired -- for my Firefox addons, configuration [about:support]; I don't want to share them, publicly.

**Comment by victoriastuart** [comment]
Edit - updated info:

The website

    https://transcranial.github.io/keras-js/

will not run (load) in Firefox started in Safe Mode (add-ons disabled; ...).  [I wanted to see if disabling add-ons helped viz-a-viz the ResNet implementation in Firefox.]

Wolf image: https://upload.wikimedia.org/wikipedia/commons/0/00/Yellowstone-wolf-17120.jpg



Inception-v3 [ImageNet]:
----

* Firefox:

    dogsled 53%
    apiary 8%
    seashore 3
    jigsaw puzzle 3%
    dowitcher 2%

* Opera:

    cairn 74%
    silky terrier 6%
    Yorkshire terrier 5%
    Australian terrier 5%
    Norwich terrier 3%

50-Layer ResNet:
----

* Firefox:

   0%
   0%
   0%
   0%
    0%

* Opera:

    timber wolf 92%
    coyote 3%
    dogsled 1%
    white wolf 1%
    red wolf 1%

**Comment by transcranial** [comment]
@victoriastuart interesting, so the resnet is problematic in firefox but inception-v3 is fine? Thanks for the report, will take a look. There are some issues with firefox I have to look into in any case.

**Comment by victoriastuart** [comment]
Yes, exactly.  Perhaps this is related: keras-js will load/process this wolf image

http://media.salon.com/2015/07/wolf.jpg

in those two nn (Inception | resnet} in Opera, but not Firefox ("Error loading URL").  So, e.g., what we (humans) see and what Opera vs. Firefox returns to keras-js may be different: underlying image format/metadata?  That might also explain the dramatic differences in classifications in the classifications of my first wolf image in those two nets, in Firefox?  Just guessing here, however.
